internal/core/logtypesapi/api.go

```diff
@@ -39,17 +38,16 @@ const LambdaName = "panther-logtypes-api"
 
 // LogTypesAPI handles the business logic of log types LogTypesAPI
 type LogTypesAPI struct {
-	NativeLogTypes func() []string
-	Database       LogTypesDatabase
-	LambdaClient   lambdaiface.LambdaAPI
-	DataCatalog    datacatalog.Client
+	NativeLogTypes    func() []string
+	Database          LogTypesDatabase
+	UpdateDataCatalog func(ctx context.Context, logType string, from, to []logschema.FieldSchema) error
+	LogTypeInUse      func(ctx context.Context) ([]string, error)
```

Comment:
LogTypeInUse -> LogTypesInUse perhaps?

Reply:
I won't change this here as this is a backport PR and I don't want to have another PR on ee just for this name. I will add a FIXME comment
```suggestion
        // FIXME: Rename to LogTypesInUse
	LogTypeInUse      func(ctx context.Context) ([]string, error)
```